recipes/sphinx-copybutton/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "sphinx-copybutton" %}
+{% set version = "0.2.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 94593d9fd7318fa119ae4144fc99a91cb5cd4523adb363b6a93ee9b2ad0160ae
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
```

Comment:
I think that could be ``noarch: python``

Reply:
thanks

Reply:
also python 3 only.

Reply:
but I think that is taken care of by flit